product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: "Managing SSL certificates"
+navTitle: "Managing SSL certificates"
+legacyRedirectsGenerated:
+  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
+  - "/edb-docs/d/edb-postgres-enterprise-manager/user-guides/administrators-guide/8.0/managing_certificates.html"
+# This file is moved from pem_admin/04_managing_certificates
+redirects:
+- /pem/latest/managing_certificates/
+- /pem/latest/pem_admin/04_managing_certificates/
+navigation:
+- replacing_ssl_certificates
+- regenerating_agent_certificates
+---
+
+PEM uses self-signed SSL certificates:
+
+- to secure requests to the [web server](#webserver-certificates) (Apache HTTPD) which provides the user interface and REST API.
+- to secure and authenticate the [PEM agent connections to the PEM backend database](#pem-backend-database-server-and-agent-connection-certificates).
+
+## Web server certificates
+
+PEM uses self-signed SSL certificates for the Apache HTTPD server. The self-signed SSL certificates and keys for the Apache HTTPD server are generated during PEM installation.
+
+To use your own SSL certificate for PEM, update the Apache HTTP configuration file (`edb-ssl-pem.conf`).
+
+Update these two SSL directives in the PEM VirtualHost section:
+
+-   `SSLCertificateFile` is your certificate file (for example, your_domain_name.crt).
+-   `SSLCertificateKeyFile` is the .key file generated when you created the CSR (for example, your_private.key).
+
+For example, make the following updates:
+
+```shell
+# Change the server name and file names in the configuration file to 
+# match your certificate files.
+SSLEngine on
+SSLCertificateFile /path/to/your_domain_name.crt
+SSLCertificateKeyFile /path/to/your_private.key
+```
+
+To increase security, you can replace the HTTPD self-signed SSL certificates with trusted third-party CA signed certificates in PEM. For more information, see [Replacing HTTPD self-signed SSL certificates](https://www.enterprisedb.com/postgres-tutorials/how-replacing-httpd-self-signed-ssl-certificates-trusted-ca-signed-certificates).
```

Comment:
```suggestion
To increase security, you can replace the HTTPD self-signed SSL certificates with trusted third-party CA signed certificates in PEM. For more information, see [Replace the server self-signed SSL certificates with trusted third-party CA certificates](#replace-the-server-self-signed-ssl-certificates-with-trusted-third-party-ca-certificates).
```
Better to link to this doc than drag in a blog article that we're not maintaining

Reply:
The link to the blog provides steps for Replacing httpd SSL certificates. And the link in our doc is for Replacing the Server SSL certificates. Two things are different, as far as I understand. We can consider adding the steps of the blog to our docs so that it can be maintained. We can discuss this and finalise it.